todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/di/PerActivity.java

```diff
@@ -10,12 +10,12 @@
 
 /**
  * In Dagger, an unscoped component cannot depend on a scoped component. As
- * {@link TasksRepositoryComponent} is a scoped component ({@code @Singleton}, we create a custom
+ * {@link AppComponent} is a scoped component ({@code @Singleton}, we create a custom
  * scope to be used by all fragment components. Additionally, a component with a specific scope
  * cannot have a sub component with the same scope.
  */
 @Documented
 @Scope
 @Retention(RetentionPolicy.RUNTIME)
-public @interface FragmentScoped {
+public @interface PerActivity {
```

Comment:
Prior app only had singleton or fragment scope. This refactor creates an additional activity scope as well. I'll change it to ActivityScoped and FragmentScoped